ublox/tests/parser_tests.rs

```diff
@@ -195,6 +195,63 @@ fn test_parse_cfg_nav5() {
     assert!(found);
 }
 
+#[test]
+#[cfg(feature = "serde")]
+fn test_esf_meas_serialize() {
+    let ret = [
+        181, 98, 16, 2, 16, 0, 243, 121, 129, 1, 24, 8, 0, 0, 77, 100, 0, 11, 211, 148, 129, 1,
+        213, 198,
+    ];
+
+    let mut parser = Parser::default();
+    let mut found = false;
+    let mut it = parser.consume(&ret);
+
+    while let Some(pack) = it.next() {
+        match pack {
+            Ok(pack) => {
+                let r = r#"{
+  "class": 16,
```

Comment:
this is kinda funky... string blocks weird looking

Reply:
You could also use the [json!](https://docs.rs/serde_json/latest/serde_json/macro.json.html) macro which creates a `serde_json::Value`. then you can do 

```rust
let expected = serde_json::json! {
    ...
};
let actual = serde_json::to_value(&pack).unwrap();
assert_eq!(expected, actual);
```

Reply:
i.e. skip dealing with the json as a string